tool.go

```diff
@@ -528,7 +528,10 @@ func main() {
 		Long: "GopherJS is a tool for compiling Go source code to JavaScript.",
 	}
 	rootCmd.AddCommand(cmdBuild, cmdGet, cmdInstall, cmdRun, cmdTest, cmdServe, cmdVersion)
-	rootCmd.Execute()
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(-1)
```

Comment:
Let's follow the behavior of the `go` tool here.

```bash
~ $ go unknowncommand
go: unknown subcommand "unknowncommand"
Run 'go help' for usage.
~ $ echo $?
2
```

So, let's use `os.Exit(2)` here.